code/_onclick/xeno.dm

```diff
@@ -96,7 +96,7 @@ Return FALSE if it didn't do anything and should count as a missed slash.
 
 If using do_afters or sleeps, use invoke_async (or manually add the relevant action delayand return FALSE
 so that it doesn't double up on the delays) so that it applies the delay immediately instead of when it finishes.**/
-/atom/proc/attack_alien(mob/user as mob)
+/atom/proc/attack_alien(mob/user as mob, dam_bonus, directional_assist_attack)
```

Comment:
Is it needed to have a "`dam_bonus, directional_assist_attack`" section on every attack_alien, even the ones not affecting mobs?

Reply:
Yes because I'm referencing variables by name when giving arguments in attack_alien() from unarmed_attack()

Reply:
The whole chain is kinda WACK